robot/EDA/resources/locators.py

```diff
@@ -104,6 +104,10 @@
         "enable_cc_warning_enabled": "//div[contains(@class, 'slds-notify') and @role='alert']/descendant::*[@data-key='warning']/../../following-sibling::span[text()='You must enable Course Connections before editing record types.']",
         "enable_cc_warning_disabled": "//span[contains(@class, 'slds-hide')]/descendant::div[contains(@class, 'slds-notify') and @role='alert']/descendant::*[@data-key='warning']/../../following-sibling::span[text()='You must enable Course Connections before editing record types.']",
     },
+    "eda_settings_system": {
+        "default_checkbox": "//span[text()='{}']/../following-sibling::div[1]/descendant::img",
+        "default_dropdown_value": "//span[text()='{}']/../following-sibling::div[1]/descendant::span[text()='{}']",
```

Comment:
Can you use something else for div instead of using indexing like eg: div[@class] or div[./span]or any other possibilities

Reply:
after speaking to rakesh, couldn't find other reasonable alternatives so decided to leave as it is